services/market/shared/market/priceUpdater.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const util = require('util');
+const BluebirdPromise = require('bluebird');
+const { Logger, CacheRedis } = require('lisk-service-framework');
+
+const config = require('../../config');
+
+const binance = require('./sources/binance');
+const bittrex = require('./sources/bittrex');
+const exchangeratesapi = require('./sources/exchangeratesapi');
+const kraken = require('./sources/kraken');
+
+const pricesCache = CacheRedis('market_prices', config.endpoints.redis);
+const logger = Logger();
+
+const supportedFiatCurrencies = ['USD', 'EUR', 'CHF'];
+
+const targetPairs = config.market.targetPairs && config.market.targetPairs.length
+	? config.market.targetPairs.split(',')
+	: [
+		'LSK_BTC', 'LSK_EUR', 'LSK_USD', 'LSK_CHF',
+		'BTC_EUR', 'BTC_USD', 'BTC_CHF',
```

Comment:
This can be defined in the config only. Or constants if validated against a fixed list.